tests/RAGchain/preprocess/text_splitter/test_token_splitter.py

```diff
@@ -0,0 +1,794 @@
+
+And tonight, I’m announcing we’re expanding eligibility to veterans suffering from nine respiratory cancers. 
+
+I’m also calling on Congress: pass a law to make sure veterans devastated by toxic exposures in Iraq and Afghanistan finally get the benefits and comprehensive health care they deserve. 
+
+And fourth, let’s end cancer as we know it. 
+
+This is personal to me and Jill, to Kamala, and to so many of you. 
+
+Cancer is the #2 cause of death in America–second only to heart disease. 
+
+Last month, I announced our plan to supercharge  
+the Cancer Moonshot that President Obama asked me to lead six years ago. 
+
+Our goal is to cut the cancer death rate by at least 50% over the next 25 years, turn more cancers from death sentences into treatable diseases.  
+
+More support for patients and families. 
+
+To get there, I call on Congress to fund ARPA-H, the Advanced Research Projects Agency for Health. 
+
+It’s based on DARPA—the Defense Department project that led to the Internet, GPS, and so much more.  
+
+ARPA-H will have a singular purpose—to drive breakthroughs in cancer, Alzheimer’s, diabetes, and more. 
+
+A unity agenda for the nation. 
+
+We can do this. 
+
+My fellow Americans—tonight , we have gathered in a sacred space—the citadel of our democracy. 
+
+In this Capitol, generation after generation, Americans have debated great questions amid great strife, and have done great things. 
+
+We have fought for freedom, expanded liberty, defeated totalitarianism and terror. 
+
+And built the strongest, freest, and most prosperous nation the world has ever known. 
+
+Now is the hour. 
+
+Our moment of responsibility. 
+
+Our test of resolve and conscience, of history itself. 
+
+It is in this moment that our character is formed. Our purpose is found. Our future is forged. 
+
+Well I know this nation.  
+
+We will meet the test. 
+
+To protect freedom and liberty, to expand fairness and opportunity. 
+
+We will save democracy. 
+
+As hard as these times have been, I am more optimistic about America today than I have been my whole life. 
+
+Because I see the future that is within our grasp. 
+
+Because I know there is simply nothing beyond our capacity. 
+
+We are the only nation on Earth that has always turned every crisis we have faced into an opportunity. 
+
+The only nation that can be defined by a single word: possibilities. 
+
+So on this night, in our 245th year as a nation, I have come to report on the State of the Union. 
+
+And my report is this: the State of the Union is strong—because you, the American people, are strong. 
+
+We are stronger today than we were a year ago. 
+
+And we will be stronger a year from now than we are today. 
+
+Now is our moment to meet and overcome the challenges of our time. 
+
+And we will, as one people. 
+
+One America. 
+
+The United States of America. 
+
+May God bless you all. May God protect our troops.""",
```

Comment:
Make this .txt file and save it tests/resource folder for ingest. 
And load text file every time you test.